DataFormats/TotemRawData/interface/TotemRawEvent.h

```diff
@@ -0,0 +1,60 @@
+/****************************************************************************
+*
+* This is a part of the TOTEM offline software.
+* Authors: 
+*   Jan Kašpar (jan.kaspar@gmail.com) 
+*    
+****************************************************************************/
+
+#ifndef _Totem_RawEvent_h_
+#define _Totem_RawEvent_h_
+
+#include <ctime>
+#include <map>
+
+/**
+ * Encapsulates meta-data (non-VFAT data) of TOTEM raw event.
+**/
+class TotemRawEvent
```

Comment:
After more careful look,
It looks like this TotemRawEvent is just a collection of headers or other global and local event identifiers.

From just the name  "XYZEvent"  and a cursory look, I took this as a container for the full event including data frames from the chips.
Maybe you could consider renaming this to TotemRawEventHeaders or TotemRawHeaders or something similarly descriptive.